src/extension.ts

```diff
@@ -159,3 +164,26 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
 // tslint:disable-next-line:no-empty
 export function deactivateInternal(): void {
 }
+
+/**
+ * This has some known limitations that will need to be addressed before removing the feature flag
+ * See https://github.com/Microsoft/vscode-azurefunctions/issues/1163
+ */
+async function getProjectTreeItems(context: vscode.ExtensionContext): Promise<AzureParentTreeItem[]> {
```

Comment:
Doesn't this deserve its own file?  Seems weird to be in the extension.ts

Reply:
Yeah I can move it